docs/pages/es/functionality/management.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: Lifecycle Management
+permalink: es/functionality/lifecycle-management
+type: functionality

```

Comment:
In type would put: _demostracion_ and also in the next documents.

Reply:
hi @rubenlorenzo  I think functionality (the valie for type) is an internal jekyll stuff, so it should not be changed. In the path `es/functionality/lifecycle-management` Iḿ not sure if it should be changed. Weŕe going to dig into #1789 to see how that plugin works, but at this moment these changes are not required.

Reply:
Agree @jesuslerma, I had in mind the comment @edwvilla. I will follow issue #1789.

Reply:
@jesuslerma I had already applied this change, Do I put in **type** _functionality_ again?

Reply:
Oh, i meant he had a typo on `demostración` didn't mean to change the jekyll type, sorry for the confusion 😅